source/viz.md

```diff
@@ -1191,76 +1257,75 @@ Histogram of Michelson's speed of light data.
 ```{index} altair; mark_rule
 ```
 
-{numref}`morley_hist` is a great start. 
-However, 
-we cannot tell how accurate the measurements are using this visualization 
+{numref}`morley_hist` is a great start.
+However,
+we cannot tell how accurate the measurements are using this visualization
 unless we can see the true value.
-In order to visualize the true speed of light, 
+In order to visualize the true speed of light,
 we will add a vertical line with the `mark_rule` function.
-To draw a vertical line with `mark_rule`, 
-we need to specify where on the x-axis the line should be drawn. 
-We can do this by creating a dataframe with just one column with value `792.458`, which is the true value of light speed 
-minus 299,000 and encoding it in the `x` axis; this ensures it is coded the same way as the 
+To draw a vertical line with `mark_rule`,
+we need to specify where on the x-axis the line should be drawn.
+We can do this by creating a dataframe with just one column with value `792.458`, which is the true value of light speed
+minus 299,000 and encoding it in the `x` axis; this ensures it is coded the same way as the
 measurements in the `morley` data frame.
-We would also like to fine tune this vertical line, 
+We would also like to fine tune this vertical line,
 styling it so that it is dashed and 1 point in thickness.
-A point is a measurement unit commonly used with fonts, 
-and 1 point is about 0.353 mm. 
-We do this by setting `strokeDash=[3,3]` and `size = 1`, respectively. 
+A point is a measurement unit commonly used with fonts,
+and 1 point is about 0.353 mm.
+We do this by setting `strokeDash=[3,3]` and `size = 1`, respectively.
 
 Similarly, a horizontal line can be plotted using the `y` axis encoding and the dataframe with one value, which would act as the be the y-intercept
 
-Note that 
-*vertical lines* are used to denote quantities on the *horizontal axis*, 
-while *horizontal lines* are used to denote quantities on the *vertical axis*. 
+Note that
+*vertical lines* are used to denote quantities on the *horizontal axis*,
+while *horizontal lines* are used to denote quantities on the *vertical axis*.
 
 To add the dashed line on top of the histogram, we will use the `+` operator. This concept is also known as layering in altair.(This is covered in the later sections of the chapter). Here, we add the `mark_rule` chart on the `morley_hist` chart of the form `mark_bar`
```

Comment:
Poor formatting of parenthetical

Also, *is* it covered later? If not, we should probably cover it here where it is first used

Reply:
honestly we should probably just cover it here anyway, because it's being used here